### DISCUSSION TITLE
What do I do when one of my dotfiles uses the same templating as chezmoi?

### DISCUSSION THREAD

**Discussion by k4y4k** [twpayne/chezmoi#1563] [Q&A] [answered]
I use `bugwarrior`. tl;dr I'd like chezmoi to leave this line alone / this is **not** a template, despite using the same syntax:

```
# before
github.description_template = #{{githubnumber}}: {{githubtitle}}

# after
github.description_template = #{{githubnumber}}: {{githubtitle}}
```

(I need that to remain a template)

but the next line **is** a template and I want chezmoi to process it as such:

```
# before 
github.token = {{ .github_token }}

# after
github.token = abc123
```

Any ideas?

**Comment by felipecrs** [comment, marked as answer]
Yes, put the template-like text inside of a template!

```helm
github.description_template = {{ "#{{githubnumber}}: {{githubtitle}}" }}
github.token = {{ .github_token }}
```

![image](https://user-images.githubusercontent.com/29582865/139372637-94eb0c5d-89ad-4820-b1dd-701049f432fd.png)

**Reply by k4y4k** [reply]
# 🤯

how did I not think of this? Thank you very much!!
